guide/english/certifications/responsive-web-design/applied-visual-design/learn-about-tertiary-colors/index.md

```diff
@@ -3,8 +3,18 @@ title: Learn about Tertiary Colors
 ---
 ## Learn about Tertiary Colors
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/responsive-web-design/applied-visual-design/learn-about-tertiary-colors/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
```

Comment:
this file is still a stub, as it only has the solution, no explanation.

Reply:
As far as I have seen, the staff has accepted PRs with no explanation as at this time , as I comprehend, We are trying to fill the guide with solutions so users don't get stuck rather than using more time and adding solutions, on which note, I request you to add the explaination

Reply:
If you don't want to add it, just leave the stub paragraph intact, as it's clearly missing.

Reply:
Can you please clarify the definition of a stub for me?
@lynxlynxlynx @raisedadead @erictleung @ezioda004 @QuincyLarson

Reply:
The Wikipedia definition is quite fitting, since it started this use:
> A stub is an article that, although providing some useful information, is too short to provide encyclopedic coverage of a subject, and that is capable of expansion.

Reply:
@thecodingaviator I'm not sure who has accepted PR's with only solutions but its not something we are suppose to.
The guide solution should atleast have some explanation and then the solution.

Reply:
All right. That seems fine!